### PR TITLE
Moved logic for container creation to cosmosdb ns

### PR DESCRIPTION
**Pull request by ahmadnazir**
@aczelandi This PR contains basic setup for DSL including a simple test and CI (on Travis). I suggest that we merge this and keep adding tests to the upcoming features.

### CODE REVIEW

File: src/wynut/azure/storage/client.clj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 (ns wynut.azure.storage.client
   (:require
-   ;; [environ.core :refer [env]]
-   [wynut.env :refer [env]]
+   [environ.core :refer [env]]
+   ;; [wynut.env :refer [env]]
```
Comment (aczelandi) [inline src/wynut/azure/storage/client.clj]: remove?

File: src/wynut/handler.clj [hunk 2]
```diff
@@ -5,28 +5,17 @@
             [prone.middleware :refer [wrap-exceptions]]
             [compojure.core :refer [defroutes GET]]
             [compojure.route :refer [not-found]]
-            [wynut.env :refer [env]]
-            [wynut.azure.storage.cosmosdb :as cosmosdb]
-            [wynut.azure.storage.azurecli :as azure]))
+            [environ.core :refer [env]]
+            ;; [wynut.env :refer [env]]
```
Comment (aczelandi) [inline src/wynut/handler.clj]: same here

File: test/wynut/azure/storage/client_test.clj [hunk 3]
```diff
@@ -2,6 +2,6 @@
   (:require [clojure.test :refer :all]
             [wynut.azure.storage.client :refer :all]))
 
-(deftest a-test
-  (testing "FIXME, I fail."
-    (is (= 0 1))))
+;; (deftest a-test
```
Comment (aczelandi) [inline test/wynut/azure/storage/client_test.clj]: guess we can delete this file

File: src/wynut/azure/storage/client.clj (hunk 1 above)
Comment (ahmadnazir) [reply src/wynut/azure/storage/client.clj]: kept it commented out as we need it while debugging.

File: src/wynut/handler.clj (hunk 2 above)
Comment (ahmadnazir) [reply src/wynut/handler.clj]: See response above.

File: test/wynut/azure/storage/client_test.clj (hunk 3 above)
Comment (ahmadnazir) [reply test/wynut/azure/storage/client_test.clj]: This is a default template and we should start adding tests for the client as well. Let's remove it in a subsequent PR.

**Pull request merged**